test/plugins/graphite_test.lua

```diff
@@ -114,12 +115,13 @@ end
 g.test_graphite_kills_previous_fibers_on_init = function()
```

Comment:
`graphite.init()` behaves differently when called multiple times:
- it creates multiple graphite workers when called if called without yields between calls
- it creates result in 1 fiber (expected behaviour) when each init call is followed by yield

I think that this will not result in hard to catch bugs but suggest to fix entire init method or at least document the difference.